packages/react/src/components/Header/Header.jsx

```diff
@@ -52,6 +52,8 @@ const propTypes = {
   appName: PropTypes.string.isRequired,
   /** Optional prop that provides additional app information */
   subtitle: PropTypes.string,
```

Comment:
If this is a one off thing for just this one product should we just open up the subtitle to allow for any node?

Reply:
I had considered this approach in the beginning, but then I realized that `SuiteHeader` would need both properties , `appName` (which is mapped to `Header`'s `subtitle`) and the `extraContent`, because `SuiteHeader` also uses `appName` independently in other places (where `extraContent` is not used). Besides, I thought it was safer to just include an optional property in an existing component (`Header`) than repurposing an existing property that might be used in many other projects that are already consuming the `Header` base component.
But if you think it is better change te PropType of `subtitle` to accept any generic element, I think I can do that.

Reply:
Ok, so, I'll try to reuse the `Header`'s `subtitle` property instead of creating the `extraContent` property. And `SuiteHeader` will have the `extraContent` to separate `appName` and the custom element. PAL `Header`'s `subtitle` property will be `oneOfType(PropTypes.string, PropTypes.element)`

Reply:
@davidicus done! I've removed `extraContent` from `Header`. Please let me know if the this approach is acceptable. Thanks.

Reply:
@blechner This change is fine as long as there is no expectation on clicking on the tag and having it be different than the title link. These will both be wrapped in the link tag and clicking on either will trigger that anchor. Which is why I suggested just keeping your original implementation in my comments 7 days ago. Sorry if that was not clear. @JordanWSmith15 do you want to confirm that this is acceptable for your use case?

Reply:
Only click event we need in the header is a "Skip to main content" link, which the header seems to already support. So, I think we are good!

Reply:
@davidicus yes, your comment was clear, but I talked with @bryancboyd and we thought that the approach to repurpose the subtitle prop would be less "invasive" to a common component that many other projects might be using, because of what you had commented earlier: "The extra prop is probably the easiest but still feels wrong if this is not a pattern we want teams to pick up."